Dockerfile

```diff
@@ -17,7 +20,7 @@ COPY ./package.json ./package-lock.json lerna.json ./
 COPY ./packages/core/package.json ./packages/core/
 COPY ./packages/sapien/package.json ./packages/sapien/
 COPY ./packages/portal-proto/package.json ./packages/portal-proto/
-RUN npm ci
+RUN npm ci --include=dev
```

Comment:
why dev deps?

Isn't this image the one we used in prod?

https://docs.npmjs.com/cli/v10/commands/npm-install

Reply:
npm ci is doing npm prepare which runs husky install and husky is in dev deps. 
There are many other dev deps (including lodash) used in lerna run --scope portal-proto build. 

Either some scripts are not well defined or some deps are not well defined. Not something we should solve here.